application-task-ui/src/main/resources/TaskManager/TaskChangedEventNotificationRenderers.xml

```diff
@@ -0,0 +1,428 @@
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="TaskManager.TaskChangedEventNotificationRenderers" locale="">
+  <web>TaskManager</web>
+  <name>TaskChangedEventNotificationRenderers</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>TaskManager.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>TaskChangedEventNotificationRenderers</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content/>
+  <object>
+    <name>TaskManager.TaskChangedEventNotificationRenderers</name>
+    <number>0</number>
+    <className>XWiki.Notifications.Code.NotificationDisplayerClass</className>
+    <guid>7a26301f-4126-44c4-9458-104592fcfce8</guid>
+    <class>
+      <name>XWiki.Notifications.Code.NotificationDisplayerClass</name>
+      <customClass/>
+      <customMapping/>
+      <defaultViewSheet/>
+      <defaultEditSheet/>
+      <defaultWeb/>
+      <nameField/>
+      <validationScript/>
+      <eventType>
+        <disabled>0</disabled>
+        <name>eventType</name>
+        <number>1</number>
+        <prettyName>Event type</prettyName>
+        <size>64</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </eventType>
+      <notificationTemplate>
+        <contenttype>VelocityCode</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>notificationTemplate</name>
+        <number>2</number>
+        <prettyName>Notification template</prettyName>
+        <restricted>0</restricted>
+        <rows>3</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </notificationTemplate>
+    </class>
+    <property>
+      <eventType>com.xwiki.task.internal.notifications.taskchanged.TaskChangedEvent</eventType>
+    </property>
+    <property>
+      <notificationTemplate>## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
```

Comment:
You don't need the license header in here, since this xml page already has one. I guess like this you are also adding more content that needs to be scrolled when looking at the file :)